addons/html_builder/static/src/utils/utils.js

```diff
@@ -34,6 +37,11 @@ export function getSnippetName(snippetEl) {
     return _t("Block");
 }
 
+export const patchSnippetModel = memoize((snippetsName, snippetModelPatch) => {
```

Comment:
Remove the util, and modify the `snippet_service` to add this function in the service API